tests/test_clients_GcpClient.py

```diff
@@ -12,6 +14,7 @@
 from google.cloud.storage import Blob
 from lib.models.Snapshot import Snapshot
 from lib.models.Volume import Volume
+from unittest.mock import Mock, patch
```

Comment:
patch is already imported [here](https://github.com/cloudfoundry-incubator/service-fabrik-backup-restore/blob/09ecd17ab00c6f7960ec3b79a3ee8c206cef2c9c/tests/test_clients_GcpClient.py#L8). Please remove.

Reply:
ok